### DISCUSSION TITLE
Suddenly: git_cliff_core::config > No releases found

### DISCUSSION THREAD

**Discussion by RWOverdijk** [orhun/git-cliff#984] [Q&A] [answered]
Hey!

Our release script stopped working due to:

> WARN  git_cliff_core::config > No releases found, using 0.1.0 as the next version.

This is super weird, because the previous 30 or so releases worked just fine. Nothing has changed in this repo, and all tags and the changelog are the same.

Did something change in git-cliff? What is it looking for but not finding?

**Comment by RWOverdijk** [comment, marked as answer]
I found the problem: I'm an idiot. I didn't merge my PR yet so there was nothing to release.

Maybe this message could be made nicer 🤔 Right now it says no releases were found, but it really means that nothing was found to release.

In any case: pebkac, sorry 😅

**Reply by orhun** [reply]
Haha, no worries! Glad to hear that you got it working :)
